package.json

```diff
@@ -36,7 +36,7 @@
     "glob-expand": "^0.2.1"
   },
   "peerDependencies": {
-    "typescript": "^2.0.6 || >=2.1.0-dev || >=2.2.0-dev"
+    "typescript": "^2.0.6 || ^2.1.5 || >=2.2.0-dev"
```

Comment:
I see, I was also wondering about this part too. But there might be a problem in how `npm` evaluates semver in peer dependency

```
npm WARN typescript-formatter@4.1.0 requires a peer of typescript@^2.0.6 || >=2.1.0-dev || >=2.2.0-dev but none was installed.
➜  ~ tsc -v
Version 2.1.4
```